docs/source/user_guide/installation/install-from-docker.rst

```diff
@@ -51,35 +33,11 @@ Once you have pulled the Docker image, you can run a Docker container with the P
 
 1. In your terminal, use the following command to start a Docker container from the pulled image:
 
-.. tab:: Basic
-
-      .. code:: bash
-
-            docker run -it pybamm/pybamm:latest
-
-.. tab:: ODES Solver
 
-      .. code:: bash
-
-            docker run -it pybamm/pybamm:odes
-
-.. tab:: JAX Solver
-
-      .. code:: bash
-
-            docker run -it pybamm/pybamm:jax
-
-.. tab:: IDAKLU Solver
-
-      .. code:: bash
-
-            docker run -it pybamm/pybamm:idaklu
-
-.. tab:: All Solver
+.. code:: bash
 
-      .. code:: bash
+    docker run -it pybamm/pybamm:latest
```

Comment:
As `:latest` is our only and default tag, user won't need to specify it explicitly for now, hence we can keep it like this - 
```suggestion
    docker run -it pybamm/pybamm
```